corehq/apps/app_manager/views/view_generic.py

```diff
@@ -194,9 +195,12 @@ def _make_name(suffix):
         'app': app,
     })
 
-    # Pass form for Copy Application to template:
+    # Pass form for Copy Application to template
+    domain_names = [d.name for d in Domain.active_for_user(request.user)]
+    domain_names.sort()
     context.update({
-        'copy_app_form': copy_app_form if copy_app_form is not None else CopyApplicationForm(app_id)
+        'copy_app_form': copy_app_form if copy_app_form is not None else CopyApplicationForm(app_id),
+        'domain_names': domain_names,
```

Comment:
it feels like these should already be in the context somewhere to render the domain switcher, no? might be nice to avoid the duplicate DB lookup and use that.